apps/core/src/pages/privacy-policy/components/DataAuthority.tsx

```diff
@@ -0,0 +1,31 @@
+import { Text, Title } from "@mantine/core";
+import { RefObject } from "react";
+import useStyles from "./general";
+
+type propType = {
+  innerRef: RefObject<HTMLDivElement>;
+};
+
+const DataAuthority = ({ innerRef }: propType) => {
```

Comment:
maybe use the `React.FC<T>` helper instead

type the props and the return val in one swoop